Documentation/deploy-master.md

```diff
@@ -301,6 +308,114 @@ spec:
       timeoutSeconds: 1
 ```
 
+### Set Up Calico Node Container
+
+The Calico node container runs on all hosts, including the master node. It performs two functions
+* It networks containers created through CNI to connect them to the flannel overlay network.
+* It enforces network policy created through the Kubernetes policy API.
```

Comment:
> It networks containers created through CNI to connect them to the flannel overlay network.

Would be nice to mention what each of these get you:

"Connects containers to the flannel overlay network, which enables the "one IP per pod" concept."

"Enforces network policy created through the Kubernetes policy API, ensuring pods talk to authorized resources only."